ModularTegustation/tegu_mobs/apocalypse_bird.dm

```diff
@@ -89,13 +100,33 @@
 /mob/living/simple_animal/hostile/megafauna/apocalypse_bird/Life()
 	. = ..()
 	if(.)
-		if((slam_cooldown <= world.time) && prob(75))
+		if(meltdown_cooldown <= world.time)
+			SSlobotomy_corp.InitiateMeltdown(round(SSlobotomy_corp.qliphoth_meltdown_amount/3)+1, TRUE)
+			meltdown_cooldown = world.time + meltdown_cooldown_time
+		if(slam_cooldown <= world.time)
 			Slam()
-		else if((judge_cooldown <= world.time) && prob(50))
-			Judge()
-		else if((light_cooldown <= world.time) && prob(50))
-			LightFire()
-		else if((teleport_cooldown <= world.time) && prob(50))
+		else if(special_cooldown <= world.time)
+			var/list/attacks = list()
+			if(judge_possible)
+				for(var/i= 1 to 25)
+					attacks += i
+			if(big_possible)
+				for(var/i = 26 to 90)
+					attacks += i
+			if(bite_possible)
+				for(var/i = 90 to 100)
+					attacks += i
```

Comment:
Why did you feel the need to do this...

Reply:
Why not just have them set to Probs with the chance being recalculated whenever an egg dies?